### ISSUE TITLE
Docker on WSL2

### ISSUE THREAD

**Issue by easterism** [DockStation/dockstation#258]
DockStation works perfectly on Linux and on Windows Home with Docker toolbox as well. But times ago I start using DockerDesktop because it works perfectly on WSL2... Now DockStation wont start anymore. Looks like no support WSL2... where can i find a reason (error log etc)?
![image](https://user-images.githubusercontent.com/2633331/94739343-85769780-0379-11eb-963b-cb1c8a8e98f7.png)

**Comment by igor-lemon** [comment]
Hi @easterism Huh, it's an interesting case. Unfortunately, I don't have Win PC with tuned WSL and I want to know 1 thing. What is the path do you have for the Docker access? By default for Windows it's `\\.\pipe\docker_engine` do you have the same path as on Linux `/var/run/docker.sock`?

**Comment by easterism** [comment]
All Win settings is default. I dont know how to check is it the same as you wrote
Linux path is /var/run/docker.sock

**Comment by igor-lemon** [comment]
Yep. A simple solution to add Docker socket path option into settings.

**Comment by elavrenchuk** [comment]
Hi @igor-lemon, I know it was long ago, but can you please check related issue #300?

**Comment by igor-lemon** [comment]
Hi @elavrenchuk 
Ok, but I need to find a Windows machine before, unfortunately, I can't check it via some VM
